readme.md

```diff
@@ -102,6 +102,7 @@
 - [RubyMotion](https://github.com/motion-open-source/awesome-rubymotion) - Write cross-platform native apps for iOS, Android, macOS, tvOS, and watchOS in Ruby.
 - [Smart TV](https://github.com/vitalets/awesome-smart-tv) - Create apps for different TV platforms.
 - [GNOME](https://github.com/Kazhnuz/awesome-gnome) - Simple and distraction-free desktop environment for Linux.
+- [Sailfish OS](https://github.com/sailfishos-community/awesome-sailfishos) - True Independent Mobile OS
```

Comment:
I think your description could be improved. This sounds like a marketing statement (and should start with 'Truly', not 'True'). Maybe: "A truly independent open-source mobile operating system."